### ISSUE TITLE
sdfc: POD document had syntax errors

### ISSUE THREAD

**Issue by sebastianbergmann** [bebbo/amigaos-cross-toolchain#38]
```
DEBUG: enter directory ".build-m68k/build/sfdc"
DEBUG: execute "make -j4"
pod2man sfdc -o sfdc.1
pod2text -c sfdc -o sfdc.txt
sfdc around line 5562: Non-ASCII character seen before =encoding in 'still�have'. Assuming CP1252
sfdc around line 5570: '=item' outside of any '=over'
sfdc around line 5598: Expected text after =item, not a bullet
sfdc around line 5604: Expected text after =item, not a bullet
sfdc around line 5608: Expected text after =item, not a bullet
sfdc around line 5614: Expected text after =item, not a bullet
sfdc around line 5621: Expected text after =item, not a bullet
sfdc around line 5626: Expected text after =item, not a bullet
sfdc around line 5635: Expected text after =item, not a bullet
sfdc around line 5640: Expected text after =item, not a bullet
sfdc around line 5645: Expected text after =item, not a bullet
sfdc around line 5649: Expected text after =item, not a bullet
sfdc around line 5653: Expected text after =item, not a bullet
sfdc around line 5657: Expected text after =item, not a bullet
sfdc around line 5662: Expected text after =item, not a bullet
sfdc around line 5562: Non-ASCII character seen before =encoding in 'still�have'. Assuming CP1252
sfdc around line 5570: =over without closing =back
POD document had syntax errors at /usr/bin/pod2text line 81.
sfdc around line 5570: '=item' outside of any '=over'
Makefile:59: recipe for target 'sfdc.txt' failed
make: *** [sfdc.txt] Error 255
make: *** Waiting for unfinished jobs....
sfdc around line 5598: Expected text after =item, not a bullet
sfdc around line 5604: Expected text after =item, not a bullet
sfdc around line 5608: Expected text after =item, not a bullet
sfdc around line 5614: Expected text after =item, not a bullet
sfdc around line 5621: Expected text after =item, not a bullet
sfdc around line 5626: Expected text after =item, not a bullet
sfdc around line 5635: Expected text after =item, not a bullet
sfdc around line 5640: Expected text after =item, not a bullet
sfdc around line 5645: Expected text after =item, not a bullet
sfdc around line 5649: Expected text after =item, not a bullet
sfdc around line 5653: Expected text after =item, not a bullet
sfdc around line 5657: Expected text after =item, not a bullet
sfdc around line 5662: Expected text after =item, not a bullet
sfdc around line 5570: =over without closing =back
POD document had syntax errors at /usr/bin/pod2man line 70.
Makefile:56: recipe for target 'sfdc.1' failed
make: *** [sfdc.1] Error 255
ERROR: command "make -j4" failed with 2
```

**Comment by bebbo** [comment]
pod2man >=5.18 ?

**Comment by bebbo** [comment]
fixed in 9afedd9d7dae74922d6456a70acf8ab917335646

**Comment by sebastianbergmann** [comment]
Verified to be fixed by 9afedd9d7dae74922d6456a70acf8ab917335646.
